### PR TITLE
EVG-18118: Display fallback project for repo patches

### PR DESCRIPTION
**Pull request by sophstad**
EVG-18118

### Description

- For patches attached to a repo instead of a project, display `owner/repo` since no project name exists. These don't link anywhere because we don't support a repo patches page.

### Screenshots

**Before:**
My patches:
image
Patch page:
image

**After:**
My patches:
image
Patch page:
image

### CODE REVIEW

**Review by ablack12 — COMMENTED** [review]
Only did a skim of the code so I might have missed it but did we also update the My Patches page view?
image

**Comment by sophstad** [comment]
> Only did a skim of the code so I might have missed it but did we also update the My Patches page view?
> image

@ablack12 Yes! I included screenshots of the My Patches page in the PR description

**Comment by ablack12** [comment]
Ah thank you sorry I pushed these together in my mind. LGTM 👍

**Review by ablack12 — APPROVED** [review]

File: src/components/PatchesPage/PatchCard.tsx [hunk 1]
```diff
@@ -49,10 +49,34 @@ export const PatchCard: React.VFC<Props> = ({
...
+    patchProject = projectIdentifier ? (
+      <StyledRouterLink
+        to={getProjectPatchesRoute(projectIdentifier)}
+        data-cy="project-patches-link"
+      >
+        <strong>{projectIdentifier}</strong>
+      </StyledRouterLink>
+    ) : (
+      `${projectMetadata.owner}/${projectMetadata.repo}`
+    );
```
Comment (khelif96) [inline src/components/PatchesPage/PatchCard.tsx:77]: Is the reasoning behind this that there is no link we can take the user too to see the rest of the patches?

**Review by khelif96 — APPROVED** [review]

File: src/components/PatchesPage/PatchCard.tsx (hunk 1 above)
Comment (sophstad) [reply src/components/PatchesPage/PatchCard.tsx:77]: Yep -- we don't currently have an equivalent repo query for a project's [Patches](https://github.com/evergreen-ci/evergreen/blob/74886a90a000ab5637663906a563a1ef60d236f5/graphql/project_resolver.go#L28-L52) resolver that could return all its patches. I think it'd be worth adding though! EVG-18197

**Pull request merged**